### DISCUSSION TITLE
Several times I need to solve bugs on GitHub by reload cookies.

### DISCUSSION THREAD

**Discussion by Mehver** [community/community#38090] [Other Feature Feedback, Questions, & Ideas]
I wonder is this my personal issue?

Two scenarios, randomly happens sometimes:

1. After I click "Generate Token" then the page jump back to the token list. No token value shows up for me to copy.
2. When I want to generate a new release, GitHub shows me the error page, and advice me to reload the page and try again.

These two problems all disappeared after I delete the current cookie of GitHub and login again.

Maybe caused by my browser? I am using the latest Chrome with many extensions. So I can't be sure if GitHub's Developers can do anything for it.

**Comment by david-engelmann** [comment]
Have you tried to reproduce the errors in a different browser? If it's only relevant to Chrome then it's probably best to submit this question somewhere Chrome specific.

**Reply by Mehver** [reply]
No, it's just a random problem I've encountered in my use. The probability is not very high, so I didn't specifically test it. I just thought maybe more people would have the same problem.
